lms/urls.py

```diff
@@ -429,6 +429,12 @@
             include('edxnotes.urls'), name="edxnotes_endpoints"),
     )
 
+    if settings.FEATURES["ENABLE_TEAMS"]:
```

Comment:
We should consider using a ``ConfigurationModel`` for the feature flag, rather than a Django setting. I'm not sure precisely what it entails, but it is the direction @cpennington is encouraging us to go.